app-backend/db/src/main/scala/MVTLayerDao.scala

```diff
@@ -44,8 +69,8 @@ object MVTLayerDao {
       z: Int,
       x: Int,
       y: Int
-  ): ConnectionIO[Option[Array[Byte]]] =
-    getAnnotationProjectTasksQ(annotationProjectId, z, x, y).option
+  ): ConnectionIO[Array[Byte]] =
+    getAnnotationProjectTasksQ(annotationProjectId, z, x, y).unique
```

Comment:
This is safe because PostGIS knows what to do with empty geometry collections

Reply:
It is not documented, however, I tested it to make sure. Here e.g. is a labels MVT query with a random UUID filled in so there won't be any matching geometries:

```sql
WITH mvtgeom as (
  SELECT
    ST_AsMVTGeom(
      annotation_labels.geometry,
      ST_TileEnvelope(11, 525, 812)
    ) AS geom,
    annotation_labels.annotation_task_id,
    annotation_label_classes.id as label_class_id,
    annotation_label_classes.name,
    annotation_label_classes.color_hex_code
  FROM
    (
      annotation_labels
      join tasks on annotation_labels.annotation_task_id = tasks.id
      JOIN annotation_labels_annotation_label_classes on annotation_labels.id = annotation_labels_annotation_label_classes.annotation_label_id
    )
    JOIN annotation_label_classes on annotation_labels_annotation_label_classes.annotation_class_id = annotation_label_classes.id
  WHERE
    ST_Intersects(
      annotation_labels.geometry,
      ST_TileEnvelope(11, 525, 812)
    )
    AND annotation_labels.annotation_project_id = '459278c2-faf1-4bf5-b010-5d423e13664f'
    AND status <> 'SPLIT'
)
SELECT
  ST_AsMVT(mvtgeom.*)
FROM
  mvtgeom;
```

It gives back `\x` which is just an empty tile

Reply:
It would have been nice if it had been documented [here](https://postgis.net/docs/ST_AsMVT.html) but this "what about empty geom collections?" question was what convinced me defining the `Monoid` would be nice, so uh ☯️ maybe

Reply:
Got it. Thanks for the example!